### ISSUE TITLE
Model architecture: Setting alpha and show affects the preview map

### ISSUE THREAD

**Issue by tephenavies** [TerriaJS/terriajs#3497]
In `master` the opacity that a layer is currently at on the main map applies to the preview map, but if the layer is hidden on the main map it's still visible on the preview map. In `mobx` both are synced between maps so the preview can be hidden.

Should all imagery layers always show and at full opacity in the preview window?

**Comment by kring** [comment]
> Should all imagery layers always show and at full opacity in the preview window?

Without necessarily having thought through all the issues, I'd say "yes".

**Comment by tephenavies** [comment]
Something like this would fix that problem:
```
/**
 * @implements Mappable
 */
class FullOpacityImageryPartsMappableAdapter {
  /**
   *
   * @param {Mappable} mappable
   */
  constructor(mappable) {
    this.mappable = mappable;
  }

  loadMapItems() {
    return this.mappable.loadMapItems();
  }

  @computed
  get mapItems() {
    return this.mappable.mapItems.map(m =>
      ImageryParts.is(m) ? { ...m, alpha: 1.0, show: true } : m
    );
  }
}
```
and then just use `new FullOpacityImageryPartsMappableAdapter(this.props.previewed)` instead of `this.props.previewed`

Or it could be fixed in TerriaViewer, but I think the above is probably better.

**Comment by tephenavies** [comment]
There are drawbacks either way. 
Implement in TerriaViewer:
- Nothing in the preview viewer can be semi transparent

Implement with adapter:
- Works for ImageryParts, might be harder for preview-specific differences for other mappables

Or you could have a custom extended TerriaViewer for the preview viewer.

**Comment by kring** [comment]
That's a cool idea. Maybe make it more of an `AdaptForPreviewMap` sort of thing, instead of full opacity / imagery parts specific? It's true that this adapter would need some details about all the types of things that might show up on a preview map, but I think that's ok.

BTW, need to be careful about making every map item full opacity, because that will make e.g. the "next" time in a time-dynamic WMS full opacity as well. Something as simple as "change to full opacity unless opacity is 0.0" could be fine.

**Comment by tephenavies** [comment]
Fixed in #3514.
